src/loki_alert_rules/error_level_logs_found.rule

```diff
@@ -0,0 +1,7 @@
+alert: ErrorLevelLogsFound
```

Comment:
IMHO, I think we could combine the error-level logs with the following fatal-level logs, and panic-level logs together. From a user perspective, those metrics are all important to monitor. Users definitely should act no matter what the log level is.

Reply:
same here, i totally agree with @wood-push-melon